src/features/appearance/index.ts

```diff
@@ -20,6 +21,15 @@ function createStyleElement() {
   document.querySelector('head')?.appendChild(styles);
 }
 
+function injectUserCustomCSS() {
+  const contentToInject = loadCustomCSS();
+  if (contentToInject !== '') {
+    const styleElement = document.createElement('style');
```

Comment:
Why can't we use the `setAppearance` function to override with the user's custom style?

Reply:
We sure can, but I think we need to change setAppearance a little so it handles a different id (I have the opinion this style should be under a different id than `custom-appearance-style`, so it is clear that comes from a user file)